frontend/src/features/entity_selector/EntitySelector.tsx

```diff
@@ -268,13 +270,31 @@ const EntitySelectorInnerAuth = ({
   };
 
   const onSlideExited = async () => {
-    setSlideDirection('up');
-    const newUid = await nextSuggestion(uid, otherUid, pollName);
-    if (newUid) {
-      onChange({ uid: newUid, rating: null });
+    if (slideDirection === 'up') {
+      const newUid = await history?.nextLeftOrSuggestion(pollName, [
+        uid,
+        otherUid,
+      ]);
```

Comment:
From a UX perspective, I am not sure it makes sense to load new suggestions on both sides of the history. I think it would be more understandable/discoverable to require a "swipe up" to get new suggestions, and to block the "swipe down" (ideally with a bounce animation) when there is no older item in the history.

What do you think?

Reply:
As discussed on Discord, we can keep this version and get feedbacks.